packages/graalvm/src/main/kotlin/elide/runtime/gvm/internals/vfs/EmbeddedGuestVFSImpl.kt

```diff
@@ -100,8 +99,8 @@ internal class EmbeddedGuestVFSImpl private constructor (
   backing: FileSystem,
   private val tree: FilesystemInfo,
   private val deferred: Boolean = Settings.DEFAULT_DEFERRED_READS,
-  private val bundles: Map<Int, BundleInfo> = TreeMap(),
-  private val knownPathMap: Map<String, VfsObjectInfo> = TreeMap(),
+  private val bundles: Map<Int, BundleInfo> = emptyMap(),
+  private val knownPathMap: Map<String, VfsObjectInfo> = emptyMap(),
```

Comment:
The previous sorted map implementation (`TreeMap`) was leaked here, replaced with proper API usage (`emptyMap()` since there no need for a specialized sorted map here)